pkg/util/informermanager/multi-cluster-manager.go

```diff
@@ -18,6 +18,9 @@ type MultiClusterInformerManager interface {
 	// The informer manager should be created before, otherwise, nil will be returned.
 	GetSingleClusterManager(cluster string) SingleClusterInformerManager
 
+	// DeleteSingleClusterResourceListerManager deletes the informer manager for a specific cluster.
+	DeleteSingleClusterManager(cluster string)
+
```

Comment:
How about doing it with another separated PR?  
Keep this PR focus on seeking a solution for better serve node status collection.

Reply:
Sure, that's a good idea. Let me modify it.